brambling/models.py

```diff
@@ -378,14 +421,32 @@ def stripe_live_can_connect(self):
     def stripe_test_can_connect(self):
         return bool(stripe_test_settings_valid() and not self.stripe_test_user_id)
 
-    def get_invites(self):
-        return Invite.objects.filter(kind=Invite.ORGANIZATION_EDITOR,
-                                     content_id=self.pk)
-
     def is_demo(self):
         return self.slug == Organization.DEMO_SLUG
 
 
+class EventMember(models.Model):
+    EDIT = '1-edit'
+    VIEW = '2-view'
+    ROLE_CHOICES = (
+        (EDIT, 'Can edit event'),
+        (VIEW, 'Can view event'),
+    )
+    event = models.ForeignKey('Event')
+    person = models.ForeignKey('Person')
+    role = models.CharField(max_length=6, choices=ROLE_CHOICES)
```

Comment:
I didn't realize that [Postgres performance for varchar & text was the same](http://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/) – that's neat! I think I'd still like to stick with a varchar, though. As much as possible, I would prefer to not tie ourselves to a particular database. I also think that CharField is more representative of what we're doing here, esp. for new general Django developers. (Really, if we were going to switch to a different field, I'd want us to investigate getting a SET implementation.)

I don't think that the limit will be an issue, since migrations are so easy. If we need to lengthen it we can. And I'm not sure that Django's TextField supports choices.